components/financial_assistance/app/controllers/financial_assistance/incomes_controller.rb

```diff
@@ -6,8 +6,8 @@ class IncomesController < FinancialAssistance::ApplicationController
 
     before_action :find_application_and_applicant
     before_action :set_cache_headers, only: [:index, :other]
-    before_action :enable_bs4_layout, only: [:other] if EnrollRegistry.feature_enabled?(:bs4_consumer_flow)
-    before_action :conditionally_enable_bs4_layout, only: [:create, :update] if EnrollRegistry.feature_enabled?(:bs4_consumer_flow)
+    before_action :enable_bs4_layout, only: [:other, :index, :create] if EnrollRegistry.feature_enabled?(:bs4_consumer_flow)
```

Comment:
does 'create' need to be declared here if it's already declared in the `conditionally_enable_bs4_layout` before_action?

Reply:
updated and removed